sdk/eventhub/event-hubs/samples/iotGetHubRuntimeInfo.ts

```diff
@@ -1,22 +1,23 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
+/*
+  This sample demonstrates how to get IoT Hub information.
```

Comment:
This sample demonstrates how to use getHubRuntimeInformation() and getPartitionInformation() to get information about the Iot Hub instance